backend/src/main/java/gov/cdc/usds/simplereport/db/model/BaseTestInfo.java

```diff
@@ -36,8 +37,8 @@ public abstract class BaseTestInfo extends AuditedEntity implements Organization
 
   @Column private Date dateTestedBackdate;
 
-  @Column
-  @Type(type = "pg_enum")
+  @Column(columnDefinition = "TEST_CORRECTION_STATUS")
+  @JdbcTypeCode(SqlTypes.NAMED_ENUM)
```

Comment:
Why was the `@type` annotation switched with `@JdbcTypeCode`?. I saw other classes that kept the `@type` so I am curious

Reply:
The previous way we were doing enums in posgres was with the [Hypersistence utils package](https://github.com/vladmihalcea/hypersistence-utils) because Posgres' uses named enums unlike other SQL dbs. [Hibernate 6.3](https://docs.jboss.org/hibernate/stable/core/javadocs/org/hibernate/type/SqlTypes.html#NAMED_ENUM) introduced [named enums](https://docs.jboss.org/hibernate/stable/core/javadocs/org/hibernate/dialect/PostgreSQLEnumJdbcType.html). The rest still have `@type` because still use the types from the Hypersistence utils package.